app/src/main/java/org/mozilla/reference/browser/BrowserApplication.kt

```diff
@@ -21,6 +23,12 @@ class BrowserApplication : Application() {
         if (isCrashReportActive) {
             components.crashReporter.install(this)
         }
+
+        System.setProperty("mozilla.appservices.fxaclient_ffi_lib_name", "reference_browser");
+        System.setProperty("mozilla.appservices.logins_ffi_lib_name", "reference_browser");
+        System.setProperty("mozilla.appservices.places_ffi_lib_name", "reference_browser");
```

Comment:
@ncalexan looks like `ReferenceBrowserComposition.init` executes the same three `setProperty` calls. Are these leftovers?

Reply:
The linter doesn't like the semicolons :).

Reply:
Yes to both.  I'm actually going to take the plugin in a different direction, which I just sketched docs (before implementing) in https://github.com/mozilla/application-services/pull/463.  Perhaps you could take a look, @csadilek?  It's pretty close to what's happening here, and shouldn't require significant changes.  It's just simpler for the plugin implementation and better for supporting unit testing transparently.

Reply:
@ncalexan read through it. Looks good to me!